kubernetes-manifests/balance-reader.yaml

```diff
@@ -41,9 +41,9 @@ spec:
           value: "8080"
         # toggle Cloud Trace export
         - name: ENABLE_TRACING
-          value: "true"
+          value: "false"
```

Comment:
This PR is for updating documentation. Are you sure that you want to bundle yaml changes with it?

Reply:
My thinking was that this change brings the manifests in sync with the documentation so they logically belonged in this PR although on their own commit.